src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java

```diff
@@ -35,10 +35,19 @@
   // for binary or string columns
   private long maxLength;
 
+  // Default value to use for min/max int when all data in the given column is NULL
+  public static final BigInteger defaultMinMaxIntVal = BigInteger.valueOf(0);
```

Comment:
Nit: public static final can be all caps. 
```suggestion
  public static final BigInteger DEFAULT_MIN_MAX_INT_VAL_FOR_EP = BigInteger.valueOf(0);
```